pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = 'setuptools.build_meta'
 [tool.black]
 line-length = 127
 skip-string-normalization = true
-target-version = ['py38', 'py37', 'py36', 'py35', 'py27']
+target-version = ['py38', 'py37', 'py36']
```

Comment:
No 3.9? Can we remove these?

Reply:
I suppose there are some reformatting rules which aren't safe to apply to older versions, like parenthesised context managers. Listing all supported versions rather than just the minimum version seems a bit pointless however.

Reply:
We're also going to have to bump black in pre-commit to add `py39` to the list, 20.8b1 does not have it.